articles/search/search-security-overview.md

```diff
@@ -166,16 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Non-personal data received from an organization, or generated from the use of the product, such as component names, will be stored and processed outside your selected region or location. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Component names include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
 
 >[!IMPORTANT]
->Object names aren't obfuscated in the telemetry logs. If possible, please avoid using names that convey sensitive information.
+>Customers shouldn't place any personal data in those fields or create applications designed to store personal data in these fields.
```

Comment:
Personal data is defined as data that can be used to identify an individual person, such as their name, address, a SSN, etc.  Hopefully no one is using PII for name fields.  

Here is the official definition -- > “Personal Data” means any information relating to an identified or identifiable natural person. An identifiable natural person is one who can be identified, directly or indirectly, in particular by reference to an identifier such as a name, an identification number, location data, an online identifier or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural or social identity of that natural person.